pysatMissions/instruments/missions_skyfield.py

```diff
@@ -0,0 +1,314 @@
+        eccentricity, mean_motion = orbits.convert_to_keplerian(alt_periapsis,
+                                                                alt_apoapsis)
+        satrec = sapi.Satrec()
+        # According to module webpage, wgs72 is common
+        satrec.sgp4init(sapi.WGS84, 'i', 0, epoch_days, bstar, 0, 0,
+                        eccentricity, np.radians(arg_periapsis),
+                        np.radians(inclination), np.radians(mean_anomaly),
+                        mean_motion, np.radians(raan))
+    else:
+        # Otherwise, use TLEs
+        satrec = sapi.Satrec.twoline2rv(line1, line2, sapi.WGS84)
+        mean_motion = satrec.mm
+
+    if one_orbit:
+        ind = times <= (2 * np.pi / mean_motion * 60)
+        times = times[ind]
+        index = index[ind]
+
+    skytimes = skyapi.load.timescale().from_datetimes(index.tz_localize('UTC'))
+
+    esat = skyapi.EarthSatellite.from_satrec(satrec, skyapi.load.timescale())
+    # Generate object with info for all times
+    sat_obj = esat.at(skytimes)
+
+    position = sat_obj.position.km
+    velocity = sat_obj.velocity.km_per_s
+
+    # Add ECEF values to instrument.
+    ecef = sat_obj.frame_xyz_and_velocity(framelib.itrs)
+
+    # Convert to geocentric latitude, longitude, altitude.
+    lat, lon = skyapi.wgs84.latlon_of(sat_obj)
+    alt = skyapi.wgs84.height_of(sat_obj).km
+
+    # Put data into DataFrame
+    data = pds.DataFrame({'position_eci_x': position[0, :],
+                          'position_eci_y': position[1, :],
+                          'position_eci_z': position[2, :],
+                          'velocity_eci_x': velocity[0, :],
+                          'velocity_eci_y': velocity[1, :],
+                          'velocity_eci_z': velocity[2, :],
+                          'position_ecef_x': ecef[0].km[0],
+                          'position_ecef_y': ecef[0].km[1],
+                          'position_ecef_z': ecef[0].km[2],
+                          'velocity_ecef_x': ecef[1].km_per_s[0],
+                          'velocity_ecef_y': ecef[1].km_per_s[1],
+                          'velocity_ecef_z': ecef[1].km_per_s[2],
+                          'geod_latitude': lat.degrees,
+                          'geod_longitude': lon.degrees,
+                          'geod_altitude': alt},
+                         index=index)
+    data.index.name = 'Epoch'
+
+    # Create metadata corresponding to variables in load routine
+    meta = pysat.Meta()
+    meta['Epoch'] = {
+        meta.labels.units: 'Milliseconds since 1970-1-1',
+        meta.labels.notes: 'UTC time at middle of geophysical measurement.',
+        meta.labels.desc: 'UTC seconds',
+        meta.labels.name: 'Time index in milliseconds'}
+    for v in ['x', 'y', 'z']:
+        meta['position_eci_{:}'.format(v)] = {
+            meta.labels.units: 'km',
+            meta.labels.name: 'ECI {:}-position'.format(v),
+            meta.labels.desc: 'Earth Centered Inertial {:}-position'.format(v),
```

Comment:
Do we want to explicitly name the reference frame here, or keep this as the type of reference frame?

Reply:
Added to notes